Library/Homebrew/cmd/postinstall.rb

```diff
@@ -27,7 +26,6 @@ def postinstall
     args.resolved_formulae.each do |f|
       ohai "Postinstalling #{f}"
       fi = FormulaInstaller.new(f)
-      fi.force = args.force?
```

Comment:
Why this change?

Reply:
It's never used inside `FormulaInstaller` for the `postinstall` step.